design.md

```diff
@@ -0,0 +1,74 @@
+Gameplay:
+
+JOINS:
+
+* User (Baylee) creates a room and joins in.
+* Baylee is places in waiting room. 
+* Baylee sends invite to others.
+* Amjith joins the room, also placed in WAITING room.
+* Princess joins the room, also placed in WAITING room.
+* Sempi joins the room, also placed in WAITING room.
+
+STARTS:
+
+* Baylee starts the game.
+* Everyone is moved to the game room.
+* Every user is asked to write a story.
+* Every user name gets a little asterisk next to it denoting we're waiting for that user to finish.
+* Princess finishes their story and submits. 
+* Princess is now in WAITING state. Asterisk is removed from Princess status.
+* Baylee and Amjith submit their stories. They both go to WAITING state.
+* Sempi goes inactive.
+* Baylee initiates next step. 
+* Amjith and Princess approve. 
+* Everyone moves to guessing state except Sempi.
+* Sempi is sent back to waiting room.
+* Everyone in the game room is presented with Baylee's story (randomly chosen).
+* Everyone submits their guesses except Baylee (no guessing on your own stories).
+* When all guesses are presented, everyone gets a reveal button when presses, shows the results.
+* Next Amjith's story is presented to everyone.
+* Baylee guesses. 
+* Princess left to get a treat before guessing.
+* Amjith initiates next step. 
+* Baylee approves. 
+* Princess is moved to waiting room.
+* Both Amjith and Baylee get a notification that Sempi wants to join the room. 
+* Either one can admit Sempi to join.
+* Amjith, Baylee and Sempi are presented with Princess' story.
+* When all of them have presented their guesses. 
+* They all get a reveal button to see the results.
+* Since there are no more stories without guesses, everyone automatically gets a writing prompt. (Results from previous round is still on top).
```

Comment:
I love the idea of this. Basically if we are WRITING that means we have no stories. One we all submit, we change the status to GUESSING and just go through the queue of stories we have, once we hit 0 it goes back to WRITING... I love it!